### PR TITLE
Refactor BootcampAdmissionsClient

### PR DESCRIPTION
**Pull request by noisecapella**
#### What are the relevant tickets?
Part of #137 

#### What's this PR do?
Code cleanup in anticipation of having BootcampAdmissionsClient use the FluidReview data

#### How should this be manually tested?
Nothing should break

### CODE REVIEW

File: klasses/bootcamp_admissions_client_test.py [hunk 1]
```diff
@@ -91,10 +92,9 @@ def test_happy_path(test_data, mocked_get_200):
     """
     Test BootcampAdmissionClient with a normal response
     """
-    user_email, url = test_data
-    boot_client = BootcampAdmissionClient(user_email)
+    user, url = test_data
+    boot_client = BootcampAdmissionClient(user)
     mocked_get_200.request.assert_called_once_with(url)
-    assert boot_client.admissions == JSON_RESP_OBJ
```
Comment (mbertrand) [inline klasses/bootcamp_admissions_client_test.py]: Add back as `assert fetch_legacy_admissions(user) == JSON_RESP_OBJ`?

File: klasses/bootcamp_admissions_client_test.py [hunk 2]
```diff
@@ -103,59 +103,52 @@ def test_happy_path(test_data, mocked_get_200):
             "is_user_eligible_to_pay": True
         }
     ]
```
Comment (mbertrand) [inline klasses/bootcamp_admissions_client_test.py]: `expected_payable_classes` is the same as `JSON_RESP_OBJ['bootcamps'][1]['klasses']`, could just use that instead here:

```python
expected_payable_klasses = JSON_RESP_OBJ['bootcamps'][1]['klasses']
```

File: klasses/bootcamp_admissions_client_test.py (hunk 1 above)
Comment (noisecapella) [reply klasses/bootcamp_admissions_client_test.py]: That's strange, must have gotten lost in a rebase

**Comment by noisecapella** [comment]
Comments addressed

**Pull request merged**